### PR TITLE
feat!: Drop python 2.7 support

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Drop python 2.7 support in favor of python 3.8

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,11 +14,14 @@ workflows:
               only: /v?[0-9]+(\.[0-9]+)*/
           matrix:
             parameters:
-              python_version: ["2.7", "3.5"]
-              django_version: ["django111", "django22"]
+              python_version: ["3.5", "3.8"]
+              debian_version: ["stretch", "buster"]
```
Comment (mariajgrimaldi) [inline .circleci/config.yml:18]: why this?

**Review by mariajgrimaldi — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (andrey-canon) [reply .circleci/config.yml:18]: that is the debian version, previous these changes the version was stretch by default and that version only supports until python 3.5  hence it's required a new debian version which supports python 3.8, besides there is no a image called 3.8-stretch  in docker hub

**Comment by magajh** [comment]
@andrey-canon are you going to add the tests for Django30,31 in a different PR?

File: .circleci/config.yml [hunk 2]
```diff
@@ -14,11 +14,14 @@ workflows:
               only: /v?[0-9]+(\.[0-9]+)*/
           matrix:
             parameters:
-              python_version: ["2.7", "3.5"]
-              django_version: ["django111", "django22"]
```
Comment (magajh) [inline .circleci/config.yml:18]: @andrey-canon are you going to add the tests for Django30,31 in a different PR?

Comment (andrey-canon) [reply .circleci/config.yml:18]: not, that is out of the scope

File: setup.py [hunk 3]
```diff
@@ -37,14 +39,13 @@ def get_version():
     packages=['eox_tenant'],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
-        'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.20',
```
Comment (mariajgrimaldi) [inline setup.py:42]: why not 2.2?

**Review by MoisesGSalas — APPROVED** [review]

**Pull request merged**